### PR TITLE
fix reload on overlay

### PR DESCRIPTION
**Pull request by Tschanzt**
@maethu @mbaechtold

### CODE REVIEW

**Comment by maethu** [comment]
👍

**Comment by mbaechtold** [comment]
I'm going to try this out. Could you please add a changelog entry in the meantime, @Tschanzt?

**Comment by mbaechtold** [comment]
Awesome, works like a charm 👍

**Comment by Tschanzt** [comment]
@mbaechtold changelog's here now.

File: ftw/referencewidget/resources/refwidget.js [hunk 1]
```diff
@@ -41,10 +41,25 @@
       widget.checkbox_template = Handlebars.compile($("#checkbox-template").html());
       widget.sel_type = widget.button.closest(".referencewidget").data("type");
 
+      widget.loadSelectedItems(widget);
+
+      $(document).on('OverlayContentReloaded', ".overlay", function(widget, event){
+          widget.button = $('#' + widget.button.attr("id"));
+          widget.loadSelectedItems(widget);
+      }.bind(null, widget));
+}
```
Comment (mbaechtold) [inline ftw/referencewidget/resources/refwidget.js]: Indentation!

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -5,6 +5,9 @@ Changelog
 1.0.2 (unreleased)
 ------------------
 
+- Reload Items when overlay is reloaded.
+  [tchanzt]
```
Comment (mbaechtold) [inline docs/HISTORY.txt]: -> tschanzt

**Comment by Tschanzt** [comment]
@mbaechtold updated.

**Comment by mbaechtold** [comment]
Cool, could you please :shipit: ?

**Pull request merged**